handbook/company/pricing-features-table.yml

```diff
@@ -959,6 +959,8 @@
       moreInfoUrl: https://osquery.readthedocs.io/en/stable/deployment/anomaly-detection/
     - description: Run a targeted YARA scan with osquery as a lightweight approach to scan anything on a host filesystem, with minimal performance impact.  Unlike full system YARA scans which consume considerable CPU resources, an equivalent YARA scan targeted in Fleet can be 8x cheaper (CPU %).
       moreInfoUrl: https://www.tripwire.com/state-of-security/signature-socket-based-malware-detection-osquery-yara
+    - description: Deploy YARA rules to your own private server that Fleet authenticates with. 
+      moreInfoUrl: https://fleetdm.com/guides/remote-yara-rules
```

Comment:
```suggestion
```

I think we can cut this because we're updating the top level URL to the guide. Also it's less to maintain if we pull it out.

Reply:
done